### PR TITLE
ActionCable README updates

### PR DESCRIPTION
**Pull request by maclover7**
[ci skip]

r? @matthewd - want to make sure I described the adapter interface correctly.

### CODE REVIEW

File: actioncable/README.md [hunk 1]
```diff
@@ -431,10 +433,7 @@ The WebSocket server doesn't have access to the session, but it has access to th
 
 ## Dependencies
 
-Action Cable is currently tied to Redis through its use of the pubsub feature to route
-messages back and forth over the WebSocket cable connection. This dependency may well
-be alleviated in the future, but for the moment that's what it is. So be sure to have
-Redis installed and running.
+Action Cable provides a subscription adapter interface to process its pubsub internals. By default, adapters for PostgreSQL, Redis, and other backends are included. To create your own adapter, you can look at `ActionCable::SubscriptionAdapter::Base` for all methods that must be implemented, and any of the adapters included within ActionCable as example implementations.
```
Comment (vipulnsward) [inline actioncable/README.md]: `and other backends are included.`, I believe we support only these two? This should be an exhaustive list for the reader.

Comment (maclover7) [inline actioncable/README.md]: We have a few like `inline`, `async`, etc but those should not really be used by end users.

Comment (kaspth) [inline actioncable/README.md]: @maclover7 async is the default adapter, so yes, it should.

Comment (maclover7) [inline actioncable/README.md]: updated. :)

**Comment by maclover7** [comment]
Merging based on :+1: from @matthewd.

**Pull request merged**